gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -675,6 +676,7 @@ public FSDataOutputStream create(
                 this, gcsPath, statistics, new CreateFileOptions(overwrite));
         break;
       case SYNCABLE_COMPOSITE:
+      case FLUSHABLE_COMPOSITE:
```

Comment:
Will it be better to put `hflush` configuration here, instead of accessing config object inside the channel?
I.e. add constructor with `boolean syncOnFlush` parameter and set it to `true`/`false` depending on the enum value? This way there will be one place where configuration depends on enum value.

Reply:
done